src/languages/en.js

```diff
@@ -872,6 +872,7 @@ export default {
             currencyInputLabel: 'Default currency',
             currencyInputHelpText: 'All expenses on this workspace will be converted to this currency.',
             save: 'Save',
+            updateNameError: 'Sorry, there was an unexpected problem updating your workspace name.',
```

Comment:
May I please have a copy check on this error message? The only time this would be shown is if there was some unexpected error when updating the workspace name. It is very unlikely.